tests/phpunit/tests/REST_API/Stories_Controller.php

```diff
@@ -208,6 +208,9 @@ public function test_get_item() {
 		$this->assertArrayHasKey( 'preview_link', $data );
 		$view_link = get_preview_post_link( $story );
 		$this->assertSame( $view_link, $data['preview_link'] );
+
+		$this->assertArrayHasKey( 'confirm_link', $data );
+		$this->assertContains( (string) $story, $data['confirm_link'] );
```

Comment:
```suggestion
		$this->assertArrayHasKey( 'embed_post_link', $data );
		$this->assertContains( (string) $story, $data['embed_post_link'] );
```

The second assertion could probably check for `post-new.php` / `?from-web-story` to be a bit more precise